docs/release_notes/2024/nov2024.md

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 1
+id: November_2024
+description: Release notes for November
+title: November 2024
+tags:
+  - release notes
+  - changelog
+  - november
+---
+
+## 3.4.1.\* (November 25, 2024)
+
+- Prophecy Python libs version: 1.9.24
```

Comment:
does this mean the latest available version? can people use earlier versions of python libraries even if they are on 3.4.1 ? (more of a "me" question than a writing question 😄 )

Reply:
[oh i found the answer ](http://localhost:3000/release_notes/version_chart/)

Reply:
It means the latest available version.